docs/en/observability/whats-new.asciidoc

```diff
@@ -21,154 +22,6 @@ https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
-
-[role="screenshot"]
-image::images/script-recorder-beta.png[Script recorder]
-
-[discrete]
-== Introducing end-to-end tracing from serverless functions with AWS Lambda support in Elastic APM (technical preview)
-
-Serverless applications are increasingly critical in modern cloud-native architectures. In 7.16 we introduced the ability to track the end-to-end performance of AWS Lambda functions as part of Elastic APM.
-
-[role="screenshot"]
-image::images/tracing-lambda.png[Trace sample]
-
-[discrete]
-== Introducing incident management workflow with Observability Alerts View (technical preview)
-
-Incident management is an important aspect of making observability data actionable. In 7.16 you can now perform alert triaging and investigation without having to use a separate user interface for this part of the workflow.
-
-[role="screenshot"]
-image::images/alerts-exp.png[Alerts]
-
-[role="screenshot"]
-image::images/alerts-view-exp.png[Alerts view]
-
-[discrete]
-== Introducing collaboration with Observability Cases (technical preview)
-
-Collaboration is an important part of troubleshooting and root cause analysis. In 7.16 Elastic Observability users can now collaborate using Cases without having to use a separate user interface for this part of the workflow.
-
-[role="screenshot"]
-image::images/alerts-case-new-exp.png[Alerts - create new case]
-
-[discrete]
-== Introducing alerts in context in APM UI (technical preview)
-
-Showing alerts with better context makes them more actionable, and enables users to start troubleshooting the issue with greater ease. With 7.16 you can click View in app on an APM alert to be taken directly to the APM UI, with the alert annotating the time series chart at the point of occurrence. Users can click the annotation in the APM UI to view further details about the alert. 
-
-Alert annotations are displayed by default in the APM UI charts at the point of their occurrence. Regardless of whether users have previously seen those alerts, the annotations help provide richer context and ready reference on alerts in the time range they are currently viewing.  
-
-[role="screenshot"]
-image::images/alerts-latency-exp.png[Alerts latency]
-
-[role="screenshot"]
-image::images/alerts-latency-graph.png[Alerts latency graph]
-
-[discrete]
-== APM failed transactions correlation charts and top values (beta)
-
-In 7.16 you can view a chart for failed transaction correlations, which shows
-the transactions in the overall latency distribution for the transaction group.
-Both the latency and failed transaction correlation tables also have a new
-option to view and add filters for the most popular field values. To learn more,
-see {kibana-ref}/correlations.html[Correlations].
-
-[role="screenshot"]
-image::images/7-16-apm-failed-correlations.png[APM failed transactions correlations]
+There are no signifant changes in {minor-version}.
```

Comment:
This is boring and not entirely true. No new features, sure, but it's a version to help people upgrade to 8.0. Whaddya think, @ollyhowell ?

Reply:
Yeah, let's go with it. If we get something later we can always add.

Reply:
```suggestion
Observability {minor-version} is a compatibility release for the {minor-version} Elastic Stack
and contains no major features.
```